layers/extras/GiftingLevel.js

```diff
@@ -0,0 +1,38 @@
+/* global cc, adv_map: true
```

Comment:
We can't have generic names for the extras ?

Reply:
There is a class called NodeExtra from which is extended for both Gifting and Bonus Level